web/src/main/java/io/fabric8/launcher/web/endpoints/GitEndpoint.java

```diff
@@ -71,8 +71,10 @@ public GitUser getUser() {
     @Secured
     @Produces(MediaType.APPLICATION_JSON)
     public Set<String> getRepositories(@QueryParam("organization") String organization) {
```

Comment:
I think you can replace the `String organization` with `ImmutableGitOrganization organization`?

Reply:
Or `GitOrganization organization` even, not sure if the Jackson serialization can handle that